rnd/autogpt_server/autogpt_server/server/server.py

```diff
@@ -93,11 +117,31 @@ def run_service(self):
             endpoint=self.get_graph,
             methods=["GET"],
         )
+        router.add_api_route(
+            path="/graphs/{graph_id}/versions",
+            endpoint=self.get_graph_all_versions,
+            methods=["GET"],
+        )
+        router.add_api_route(
+            path="/graphs/{graph_id}/versions/{version}",
+            endpoint=self.get_graph,
+            methods=["GET"],
+        )
```

Comment:
Idea: add an endpoint

    GET /graph/{graph_id}/versions/{graph_version}/executions

which would do the same as

    GET /graph/{graph_id}/executions?graph_version={graph_version}